CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: Spike Ravine - Collecting the pickup from below with High Jump Boots, Super Jump (Expert), and Unmorph Extend (Advanced).
 
+##### Area 4 Central Caverns
+
+- Added: Out of bounds movement (Intermediate) with Space Jump to access Crystal Mines.
```

Comment:
- Added: Transport to Area 3 and Crystal Mines: A beginner Spider Ball Clip to reach the elevator to Crystal Mines from the bottom Chozo Seal with Space Jump. 

We always want to mention the room name and ideally the connection.